CHANGELOG.md

```diff
@@ -28,6 +28,13 @@
 
   ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
+- Fix error summary linking on the example pages
```

Comment:
We don't need to add a CHANGELOG since these are review pages for maintainers (us and you) not public guidance. :+1:

Reply:
Ah ha, good to know. Thanks.